nesta/packages/magrxiv/collect_magrxiv.py

```diff
@@ -0,0 +1,45 @@
+from nesta.packages.mag.query_mag_api import get_journal_articles 
```

Comment:
this is actually the same as the `biorxiv_collect` that you previously reviewed, but I've renamed to reflect the fact that it now picks up bio and medrxiv articles